pkg/operator/kube_cloud_config/bootstrap.go

```diff
@@ -0,0 +1,86 @@
+package kube_cloud_config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/yaml"
+
+	configv1 "github.com/openshift/api/config/v1"
+	operatorclient "github.com/openshift/cluster-config-operator/pkg/operator/operatorclient"
+)
+
+// ValidateFile verifies a file exists, has content, and is a regular file
+func ValidateFile(path string) error {
+	st, err := os.Stat(path)
+	if err != nil {
+		return err
+	}
+	if !st.Mode().IsRegular() {
+		return fmt.Errorf("%s is not a regular file.", path)
+	}
+	if st.Size() <= 0 {
+		return fmt.Errorf("%s is empty.", path)
+	}
+	return nil
+}
+
+func BootstrapTransformer(infrastructureFile string, cloudProviderFile string) ([]byte, error) {
```

Comment:
go doc. What does this do?

BootstrapTransformer is  not a verb, but it should. It generates a target cloud config? Call it like that.

Reply:
Renamed to BootstrapTransform() and added comment block at top of function.